dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -199,6 +199,7 @@ if [ -z "${OPENSHIFT_TOKEN+x}" ]; then
   OPENSHIFT_USERNAME=${OPENSHIFT_USERNAME:-${DEFAULT_OPENSHIFT_USERNAME}}
   OPENSHIFT_PASSWORD=${OPENSHIFT_PASSWORD:-${DEFAULT_OPENSHIFT_PASSWORD}}
 fi
+DEFAULT_CHE_INFRA_OPENSHIFT_MASTER__URL="${OPENSHIFT_ENDPOINT}/auth/"
```

Comment:
I don't think that `/auth` is really should be here. It can make an impression that it is auth endpoint, but actually, this URL is a parameter for creating fabric8 OpenShift client. Maybe it will work with `auth` in the path, but `"${OPENSHIFT_ENDPOINT}"` should be enough. 
Another note about what will happen if we don't specify this parameter -> fabric8 OpenShift client will use data from mounted service account token. I think it's kind of the same if we specify `"${OPENSHIFT_ENDPOINT}"` for master URL. So I'm OK to add default value for it to make it more obvious.

Reply:
Updated so that `export CHE_INFRA_OPENSHIFT_MASTER__URL=${CHE_INFRA_OPENSHIFT_MASTER__URL:-${OPENSHIFT_ENDPOINT}}` in the ocp.sh script. Removed from deploy_che.sh.